test/unit/utils/model-utils.test.ts

```diff
@@ -0,0 +1,54 @@
+import type { Sequelize } from '@sequelize/core';
+import { isModelStatic, isSameInitialModel, Model } from '@sequelize/core';
+// eslint-disable-next-line import/order
+import { expect } from 'chai';
+
+const sequelize: Sequelize = require('../../support').sequelize;
+
+describe('isModelStatic', () => {
+  it('returns true for model subclasses', () => {
+    const MyModel = sequelize.define('myModel', {});
+
+    expect(isModelStatic(MyModel)).to.be.true;
+  });
+
+  it('returns false for model instances', () => {
+    const MyModel = sequelize.define('myModel', {});
+
+    expect(isModelStatic(new MyModel())).to.be.false;
+  });
+
+  it('returns false for the Model class', () => {
+    expect(isModelStatic(Model)).to.be.false;
+  });
+
+  it('returns false for the anything else', () => {
+    expect(isModelStatic(Date)).to.be.false;
+  });
+});
+
+describe('areSameInitialModel', () => {
```

Comment:
```suggestion
describe('isSameInitialModel', () => {
```

Since that's the name of the function we are testing

Reply:
I thought I reverted that :x